wazo_acceptance/steps/user.py

```diff
@@ -200,3 +200,17 @@ def then_user_has_no_unconditional_forward(context, firstname, lastname, forward
     confd_user = context.helpers.confd_user.get_by(firstname=firstname, lastname=lastname)
     forward = context.confd_client.users(confd_user).get_forward(forward_name)
     assert_that(forward['enabled'], is_(False))
+
+
+@when('"{firstname} {lastname}" does a blind transfer to "{exten}@{exten_context}" with API')
```

Comment:
Would it be possible to extract the transfer type? Or would that make the step unreadable?

Reply:
I don't know, For this branch I don't want implement something we do not use.
But keep your comment for https://github.com/wazo-platform/wazo-acceptance/pull/64
There are a tiny difference, but I must admit that I'm not convinced to merge steps between them. To me we reach the limit of readability vs duplication ... Since I don't really want to break the convention to put double quote on variable, I think it will be less readable